scripts/proposeStreamVOTE.js

```diff
@@ -0,0 +1,74 @@
+const hre = require("hardhat");
+
+async function main() {
+  const {
+    TAU_PER_STREAM,
+    SCHEDULE_START_TIME,
```

Comment:
We could remove these to avoid mistakes and keep track of the parameters used for in this deployment.
Is TAU 0 for VOTE tokens ?

Reply:
No, it uses the default value, will change it to zero if it is a requirement.